packages/components/src/avatar/src/Avatar.css

```diff
@@ -44,6 +44,12 @@
     transition: background-color var(--o-ui-easing-duration-2) var(--o-ui-easing-productive);
 }
 
+/* DELETED / ANONYMOUS */
+.o-ui-avatar svg {
```

Comment:
you can probably add a class to the svgs instead of relying on the DOM element of the icon. 

add a className of o-ui-avatar-anonymous-img or something like that